src/Core/Localization/CLDR/Reader.php

```diff
@@ -592,6 +592,24 @@ protected function mapLocaleData(SimpleXMLElement $xmlLocaleData, $supplementalD
             $localeData->setCurrencies($currencies);
         }
 
+        // Territories
+        $localeDisplayNamesData = $xmlLocaleData->localeDisplayNames;
+        if (isset($localeDisplayNamesData->territories)) {
+            $territories = [];
+            foreach ($localeDisplayNamesData->territories->children() as $territoryNode) {
+                $territoryCode = (string) $territoryNode->attributes()->type;
+                $variant = (string) $territoryNode->attributes()->variant;
+                if (!empty($territories[$territoryCode]) && empty($variant)) {
```

Comment:
You should add a comment explaining why $variant can't be empty :thinking:

Reply:
Fixed